components/nexus-core/src/main/java/org/sonatype/nexus/internal/log/DefaultLogManagerMBean.java

```diff
@@ -1,113 +0,0 @@
-/*
- * Sonatype Nexus (TM) Open Source Version
- * Copyright (c) 2008-2015 Sonatype, Inc.
- * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
- *
- * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
- * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
- *
- * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
- * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
- * Eclipse Foundation. All other trademarks are the property of their respective owners.
- */
-package org.sonatype.nexus.internal.log;
-
-import java.io.IOException;
-
-import javax.management.StandardMBean;
-
-import org.sonatype.nexus.log.LogConfiguration;
-import org.sonatype.nexus.log.LogManager;
-import org.sonatype.nexus.log.LogManagerMBean;
-import org.sonatype.nexus.log.LoggerLevel;
-
-import com.google.common.base.Preconditions;
-
-/**
- * Default {@link LogManagerMBean}.
- *
- * @since 2.1
- */
-public class DefaultLogManagerMBean
-    extends StandardMBean
-    implements LogManagerMBean
```

Comment:
FTR this was removed as Logback has its own JMX integration.  If we want to bring back more customized JMX for logging we can use nexus-jmx for that.